kalite/topic_tools/__init__.py

```diff
@@ -491,10 +491,35 @@ def get_assessment_item_data(request, assessment_item_id=None):
     if not assessment_item:
         return None
 
-    # TODO (rtibbles): Enable internationalization for the assessment_items.
+    # Enable internationalization for the assessment_items.
+    try:
+        item_data = json.loads(assessment_item['item_data'])
+        question_content = _(item_data['question']['content'])
+        answerarea_content = item_data['answerArea']['options']['content']
+        if answerarea_content != "":
```

Comment:
Also, why do you expect that this item may be empty, but the others (like `item_data['question']['content']`) are not?